src/pages/_document.tsx

```diff
@@ -23,11 +23,8 @@ export default class MyDocument extends Document<{ Decorator: Components }> {
       env: serverEnv.DECORATOR_ENV,
       context: isAudienceSykmeldt ? "privatperson" : "arbeidsgiver",
       chatbot: true,
-      enforceLogin: serverEnv.MOCK_BACKEND !== "true",
+      redirectToApp: true,
```

Comment:
Hmz, støtter dekoratøren tokenx/wonderwall? Blir det riktig at den skal vise utloggingsvarsel, gjøre redirects osv?

Reply:
..trodde redirecttoapp tok man via loginservice

Reply:
Dekoratøren klarer dessverre ikke å håndtere session av idporten/wonderwall mtp på utloggingsvarsel, man kastet ut. 

Hm, godt spørsmål :P Det funket å bruke det parameteret med idporten/wonderwall da vi testet, for å komme til riktig url for AG og ikke komme tilbake til nav.no. Usikker på hvorfor det funker.